### DISCUSSION TITLE
Utilizing Gradle

### DISCUSSION THREAD

**Discussion by TheRealRyGuy** [jdbi/jdbi#2358] [Ideas]
Has Gradle been considered for use with this project? Parallel building this many modules with caches would probably drastically improve build time, and I'd be more than down to port the bulk of it over

**Comment by spannm** [comment]
The project's fared well with Maven and members are used to and experienced in it.
If build times are a concern, consider `mvnd` or `mvn --threads <threadCount>` (or upgrading hardware)
We also offer a profile `fast` which skips time consuming steps such as test execution and static analyses (compile only).
